library/network/src/lib/y2firewall/firewalld/zone_parser.rb

```diff
@@ -70,6 +70,8 @@ def parse
           end
         end
 
+        zones.map { |z| z.modified = [] }
```

Comment:
this breaks encapsulation a bit. Something like ```z.reset``` looks better to me

Reply:
MM, not sure if reset at all.. because maybe reset could mean other logic. Basically the thing is that when are calling the setter method we mark the zone as modified, but in this case it shouldn't as it is basically a complete read.

Reply:
well I don't care about name much. I'm just against setting ```z.modified = []```

Reply:
Added untouched! method to reset modifications